pkg/connectors/datacatalog/openapiclient/.openapi-generator/FILES

```diff
@@ -0,0 +1,13 @@
+.gitignore
```

Comment:
Does this file need to be committed?

Reply:
This file is now removed from the commit. Please check. Thanks.